tests/contrib/pylons/test_pylons.py

```diff
@@ -659,7 +660,7 @@ def test_pylons_body_xml(self):
             response = self.app.post(
                 url_for(controller="root", action="body"),
                 params=payload,
-                extra_environ={"CONTENT_TYPE": "application/xml"},
+                extra_environ={"CONTENT_TYPE": "application/xml; charset=euc-jp"},
```

Comment:
I am not sure I love that we are tossing these into other tests randomly rather than having dedicated tests for this, but it gets the job done for now.

Reply:
Sorry, these are the start of other interesting corner cases to try, but they require some more time to do them properly. They are harmless, but I can remove them if they could be misleading.